product_docs/docs/pgd/5/admin-manual/installing/07-configure-proxies.mdx

```diff
@@ -102,6 +102,16 @@ CREATE USER pgdproxy PASSWORD 'proxysecret';
 GRANT bdr_superuser TO pgdproxy;
 ```
 
+Set node level route_dsn option for each node with database user pgdproxy. The proxy uses these dsn while connecting to database nodes. Please note the initial dsn (on proxy start) is taken from proxy config file (see below). So, ideally, route_dsn given in this section and in config file should match.
```

Comment:
This is a new tangible step and needs to be called out as such or it needs to be folded into the creation of the proxies.

It also needs to be replicated in the steps at the top of this page which serve as the general reference to the process. This is currently only in the worked example.

Reply:
Yes, but added it in same section, so, it is clear it is required only if someone is creating separate user for proxy i.e. step mentioned in section `## Create a pgdproxy user on the database`. So, kept them together. But yes please change it as you find suitable. My only intention was to add the step which is required if someone decides to go with separate user (i.e. altering route_dsn part).